### PR TITLE
[macOS] Troubleshoot Installer

### PR DESCRIPTION
**Pull request by michaelstingl**
- Added macOS installer logging
- Moved with MSI installer logging to Troubleshooting docs

@EParzefall @mmattel please review and polish if needed

Can be back ported.

### CODE REVIEW

**Review by erikjv — APPROVED** [review]

**Comment by mmattel** [comment]
I will check asap.

File: modules/ROOT/pages/appendices/troubleshooting.adoc [hunk 1]
```diff
@@ -224,6 +224,29 @@ QT_LOGGING_RULES='*=false;sync.httplogger=true' \
   --logdebug --logfile <file>
 ----
 
+=== Installer Log files
```
Comment (EParzefall) [inline modules/ROOT/pages/appendices/troubleshooting.adoc:227]:
```suggestion
=== Installer Log Files
```

File: modules/ROOT/pages/appendices/troubleshooting.adoc [hunk 2]
```diff
@@ -224,6 +224,29 @@ QT_LOGGING_RULES='*=false;sync.httplogger=true' \
...
+==== Windows Installer Logging
+
+In case you experience issues, you can run the installer with logging enabled:
+
+[source,console,subs="attributes+"]
+----
+msiexec /i ownCloud-{client-version-win}.msi /L*V "C:\log\example.log"
+----
+
+See the: https://www.advancedinstaller.com/user-guide/qa-log.html[How do I create an installation log] documentation for more information about the Msiexec.exe command and logging.
```
Comment (EParzefall) [inline modules/ROOT/pages/appendices/troubleshooting.adoc:238]: Looks like Microsoft uses the spelling with all lower case: msiexec.exe, at least on English websites.

**Review by EParzefall — COMMENTED** [review]
Some language issues

File: modules/ROOT/pages/appendices/troubleshooting.adoc (hunk 2 above)
Comment (mmattel) [inline modules/ROOT/pages/appendices/troubleshooting.adoc:238]:
```suggestion
See the: https://www.advancedinstaller.com/user-guide/qa-log.html[How do I create an installation log] documentation for more information about the msiexec.exe command and logging.
```

Comment (mmattel) [reply modules/ROOT/pages/appendices/troubleshooting.adoc]: fixed to lower case

**Comment by mmattel** [comment]
I will check image sizes before merging.

**Review by mmattel — APPROVED** [review]

**Pull request merged**